### ISSUE TITLE
ARM64 support for UWP (Particularly useful for Hololens 2)

### ISSUE THREAD

**Issue by havokentity** [microsoft/winrtc#147]
**Is your feature request related to a problem? Please describe.**
I am extremely frustrated that I need to send a compressed texture over a network protocol manually at < 10 fps to play a video
stream on the Hololens 2

**Describe the solution you'd like**
Easy build support for ARM64 UWP which is the Hololens 2 platform

**Describe alternatives you've considered**
Manually sending still frames to my Hololens 2

**Additional context**
None

**Comment by ragavendranbalaCES** [comment]
Hi @havokentity , does it support ARM64 in Unity? and if possible can you share your code snippet which you're using for sending the video?

**Comment by havokentity** [comment]
> Hi @havokentity , does it support ARM64 in Unity? and if possible can you share your code snippet which you're using for sending the video?

Yeah it works but I've since used something else to get 60 fps for video streaming. Don't remember what I did and where the code is but I think I did something with Unity/Burst etc

**Comment by ragavendranbala** [comment]
@havokentity , how you built the c++ plugin for uwp unity? Where you find the documentation for APIs? Can you share some reference to do it?
